src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/Invariant.cs

```diff
@@ -241,13 +241,14 @@ private static bool IsDialogOverrideEnabled
                 object dbgJITDebugLaunchSettingValue;
                 string dbgManagedDebuggerValue;
                 key = Registry.LocalMachine.OpenSubKey("Software\\Microsoft\\.NETFramework");
-                dbgJITDebugLaunchSettingValue = key.GetValue("DbgJITDebugLaunchSetting");
-                dbgManagedDebuggerValue = key.GetValue("DbgManagedDebugger") as string;
                 //
                 // Check for the enable.
                 //
                 if (key != null)
                 {
+                    dbgJITDebugLaunchSettingValue = key.GetValue("DbgJITDebugLaunchSetting");
+                    dbgManagedDebuggerValue = key.GetValue("DbgManagedDebugger") as string;
```

Comment:
```suggestion
                    object dbgJITDebugLaunchSettingValue = key.GetValue("DbgJITDebugLaunchSetting");
                    string dbgManagedDebuggerValue = key.GetValue("DbgManagedDebugger") as string;
```

Reply:
Can you merge the declaration here and remove the declarations from Line 241-242